### ISSUE TITLE
new steam update breaks layout

### ISSUE THREAD

**Issue by infinitephantasm** [jonnyboy0719/Flat-Green-Steam#8]
![image](https://user-images.githubusercontent.com/23409983/43178650-6b583fbe-8f9c-11e8-9f72-dd9074610dfc.png)

noticable changes:
- topright/steam avatar + friends button is completely fucked, not sure if this can be changed
- items complete misadjusted
- maybe add a game button is also misadjusted?
- address bar is awkward
![image](https://user-images.githubusercontent.com/23409983/43178681-9a4e6eb0-8f9c-11e8-8b1f-8e2a09527ee3.png)

other than those, things SEEM fine? although a fix to this would be wonderful. the friends list obviously can't be reskinned now, but resolving this is a nice step

**Comment by Allanbuzzy** [comment]
Yeah I have these issues also. I do believe that there would be a fix, but I honestly don't know. I would tinker around with some of the layout scripts but I can't really tell which ones causing each issue.
